scripts/test.sh

```diff
@@ -25,17 +24,13 @@ if [ $# -ne $EXPECTED_ARGS ]; then
     DO_showUsage
 fi
 
-DO_testLinux() {
+DO_testDocker() {
     echo ""
-    echo "######################"
-    echo "## Testing on Linux ##"
-    echo "######################"
+    echo "#######################"
+    echo "## Testing in Docker ##"
+    echo "#######################"
     echo ""
-    mkdir -p target
-    if [ ! -f target/chromedriver ]; then
-        curl -sS $lin_url > target/driver.zip && unzip target/driver.zip && rm target/driver.zip && mv chromedriver target/
-    fi
-    lein with-profile linux test
+    docker build -t fc/fcuitest .
```

Comment:
don't cache the image. Since the image won't  "change" it wouldn't be completely rebuild each time, and test may not really "run" 

```
docker build --no-cache=true -t fc-fcuitest .
```